guides/source/getting_started.md

```diff
@@ -1996,7 +1995,7 @@ Security
 If you were to publish your blog online, anyone would be able to add, edit and
 delete articles or delete comments.
 
-Rails provides a very simple HTTP authentication system that will work nicely in
+Rails provides an HTTP authentication system that will work nicely in
```

Comment:
Still think this should be `a`, no?

Reply:
"simple" here is "modest/lacking/limited", not "easy"

Reply:
`HTTP` is spelt with a consonant but actually starts with an `A` vowel sound, e.g. `an apple`, compare that to consonant sound, e.g. `a car`. So `an HTTP` is correct.

Reply:
@matthewd Oh, those are nice synonyms. How would you feel if I replaced "simple" with "modest" here?